receiver/jmxreceiver/receiver.go

```diff
@@ -259,3 +281,16 @@ func (jmx *jmxMetricReceiver) buildJMXMetricGathererConfig() (string, error) {
 
 	return strings.Join(content, "\n"), nil
 }
+
+func parsePasswordFile(path string) (map[string]string, error) {
```

Comment:
While storing the password shouldn't we be salting, and hashing the password for security?

Reply:
Oh gotcha, that seems like security risk but I guess not on our end. Curious on what app-sec thinks?